cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -1650,28 +1649,28 @@ elaborateInstallPlan
               -- Once you've implemented this, swap it for the code below.
               cuz_buildtype =
                 case PD.buildType (elabPkgDescription elab0) of
-                  PD.Configure -> cuz "build-type is Configure"
-                  PD.Custom -> cuz "build-type is Custom"
+                  PD.Configure -> [CuzBuildType CuzConfigureBuildType]
+                  PD.Custom -> [CuzBuildType CuzCustomBuildType]
                   _ -> []
```

Comment:
What about `PD.Make`?

Reply:
It does seem that PD.Make should also cause us to build legacy fallback. Updating...